examples/computer_vision/openimage-detect.py

```diff
@@ -4,7 +4,7 @@
 from pydantic import BaseModel
 
 from datachain import C, DataChain, File
-from datachain.sql.functions import path
+from datachain.lib.func import path
```

Comment:
[Q] Do we want to import `func` from datachain instead?

Reply:
** same for all examples - I don't think it matters which approach we take but consistency between examples would be good

Reply:
right. `lib` needs to be deprecated. User should import like `from datachain.func import path`

Reply:
Agree! I was thinking to move `func` module to the top-level (into `datachain` directory) to have more "plain" structure.

Reply:
Done! 👍